sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -25,6 +25,7 @@ ctypedef fused integral:
 
 ctypedef np.float64_t DOUBLE
 
+
 def csr_row_norms(X):
```

Comment:
This method should better be named `csr_row_squared_norms` (or `squared_csr_row_norms`) but let's not change this now.